subsys/net/ip/ipv4.c

```diff
@@ -147,7 +151,16 @@ int net_ipv4_parse_hdr_options(struct net_pkt *pkt,
 				return -EINVAL;
 			}
 
+			if (opt_len < 2U) {
+				return -EINVAL;
+			}
+
 			opt_len -= 2U;
+
```

Comment:
remove the empty line

Reply:
opt_len -= 2U;
are you asking to remove this empty line?
                        if (opts_len < 1U) {
                                return -EINVAL;
                        }

Reply:
ah indeed opt_len/opt**S**_len, confused the 2

Well maybe you could move the line 158 after line 162 then.

Reply:
Ok

Reply:
@tbursztyka  updated, take a look.

Reply:
> ah indeed opt_len/optS_len, confused the 2

it's never too late to rename it (but preferrably in a separate commit).